src/components/Coverage/Coverage.js

```diff
@@ -78,11 +79,17 @@ export default class Coverage extends React.Component {
 
     if (isMonograph) {
```

Comment:
Can you use declarative approach here using the ternary operator `?` So

`return isMonograph ? (...) : (...)`